packages/admin-app/src/pages/ElectionListPage.tsx

```diff
@@ -1,6 +1,115 @@
-import { Grid } from '@material-ui/core';
-import React from 'react';
+import { ApiClientFactory, Election } from '@electionguard/api-client';
+import { Grid, Container, makeStyles } from '@material-ui/core';
+import { DataGrid, GridColDef } from '@material-ui/data-grid';
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import FilterToolbar from '../components/FilterToolbar';
+import GoHomeButton from '../components/GoHomeButton';
+import InternationalText from '../components/InternationalText';
 
-export const ElectionListPage: React.FC = () => <Grid>Election List</Grid>;
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 0.75,
+    },
+    title: {
+        fontSize: 40,
+        textAlign: 'center',
+        display: 'block',
```

Comment:
This `block` display seems odd. I would just put a Flex in the container and then center everything in that. That will guarantee the full width without all of these pieces to center.